clang/lib/AST/ExprConstant.cpp

```diff
@@ -14683,6 +14710,23 @@ bool FloatExprEvaluator::VisitCallExpr(const CallExpr *E) {
   default:
     return false;
 
+  case Builtin::BI__builtin_frexp:
+  case Builtin::BI__builtin_frexpf:
+  case Builtin::BI__builtin_frexpl: {
```

Comment:
`__builtin_frexpf16` and `__builtin_frexp128` should probably be added at the same time as the other `__builtin_*` cases.

Reply:
You mean adding :
`case Builtin::BI__builtin_frexp16:`
`case Builtin::BI_builtin_frexp128:`
to the cases here?

Reply:
Done.